### PR TITLE
Persist ETH1 Information

### LINKED ISSUE #1430: Store Previously Requested ETH1.0 Logs Block Number
Hi all,

Currently our powchain service reads all historical logs from the deposit contract from ETH1.0. As the contract grows even more, we should be persisting the last requested block number for logs to disk in the powchain service to avoid looping through all of these logs every time the node is launched. a PR for this would store the last requested ETH1.0 block number for log fetching in BoltDB. Next time the node spins up, we can request logs started from this block number + 1. Thanks @nisdas for bringing this up.

### PR DESCRIPTION
**Pull request by nisdas**
Part of #1430
`Eth1ChainData` is used represent all our eth1 chain data so that it can be serialized for persistent storage.
- [x] Add separate data structures of what to persist in the db. 
- [x] Generate proto file for those structures

### CODE REVIEW

File: proto/beacon/db/powchain.proto [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    LatestETH1Data current_eth1_data = 1 ;
+    ChainStartData chainstart_data = 2;
+    SparseMerkleTrie trie = 3;
+}
+
+// LatestETH1Data contains the current state of the eth1 chain.
+message LatestETH1Data {
+    uint64 block_height = 2;
+    uint64 block_time = 3;
+    repeated bytes block_hash = 4;
```
Comment (terencechain) [inline proto/beacon/db/powchain.proto]: do we need ssz-size tag for this?

**Comment by terencechain** [comment]
Can you please add the purpose of the Eth1ChainData message in the description?

File: proto/beacon/db/powchain.proto (hunk 1 above)
Comment (nisdas) [reply proto/beacon/db/powchain.proto]: we do not need it, we are just using protobufs to serialize this for disk storage.

**Comment by nisdas** [comment]
Added in a comment on it @terencechain

File: beacon-chain/db/kv/powchain.go [hunk 2]
```diff
@@ -0,0 +1 @@
+package kv
```
Comment (prestonvanloon) [inline beacon-chain/db/kv/powchain.go]: What's up with this?

Comment (rauljordan) [reply beacon-chain/db/kv/powchain.go]: +1 on this package name, it can be specified in the .proto file

Comment (nisdas) [reply beacon-chain/db/kv/powchain.go]: removed it

**Pull request merged**